### PR TITLE
Fix Content on Startups Program Page

### PR DESCRIPTION
**Pull request by ItzNotABug**
## What does this PR do?

Minor fixes.

## Test Plan

Manual.

## Related PRs and Issues

N/A.

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes.

### CODE REVIEW

File: static/images/community/avatars/7.png
Comment (loks0n) [inline static/images/community/avatars/7.png:1]: Can we resize these images? It is very large compared the space it takes up as an icon in the page

Comment (loks0n) [reply static/images/community/avatars/7.png:1]: It should probably be less than 20KB

Comment (loks0n) [inline static/images/community/avatars/7.png:1]: Can we give this a better name?

**Review by loks0n — APPROVED** [review]

File: static/images/community/avatars/7.png
Comment (ItzNotABug) [reply static/images/community/avatars/7.png:1]: What can we name it? `walter`, `demo`, `user`, `customer`, etc?

Comment (loks0n) [reply static/images/community/avatars/7.png:1]: Haha this can be the face of walter for sure... `walter.png`?

Comment (ItzNotABug) [reply static/images/community/avatars/7.png:1]: Done!

**Review by loks0n — APPROVED** [review]

**Review by TorstenDittmann — APPROVED** [review]

**Pull request merged**